x-pack/legacy/plugins/siem/public/containers/case/use_delete_cases.tsx

```diff
@@ -57,9 +58,10 @@ const dataFetchReducer = (state: DeleteState, action: Action): DeleteState => {
       return state;
   }
 };
+
 interface UseDeleteCase extends DeleteState {
   dispatchResetIsDeleted: () => void;
-  handleOnDeleteConfirm: (caseIds: string[]) => void;
+  handleOnDeleteConfirm: (caseIds: DeleteCase[]) => void;
```

Comment:
I am wondering what is the advantage to create a new type, I am sure that you thought of it. Maybe we can do a Pick or just use Case interface

Reply:
`Pick` works

Reply:
discussed reasoning in slack, its remaining as is